src/utils/getVisiblePeople.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  Person, FiltersParams, SortType, SortDirection,
+} from '../types';
+
+export function getVisiblePeople(
+  people: Person[],
+  {
+    query, centuries, sex, sort, order,
+  }: FiltersParams,
+) {
+  let visiblePeople = [...people];
```

Comment:
not need to create copy, because u use filter method everywhere

Reply:
..but I'm using `sort` in line 36